src/main/java/com/google/code/beanmatchers/ListGenerator.java

```diff
@@ -0,0 +1,11 @@
+package com.google.code.beanmatchers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class ListGenerator implements ValueGenerator<List> {
+
+  public List generate() {
+    return new ArrayList();
```

Comment:
`ValueGenerator` requires repeated calls to the generator to return items that have are not equal per the `equals` method. Violating this introduces #21 .